deploy/kourier-knative.yaml

```diff
@@ -16,6 +16,14 @@ metadata:
   namespace: kourier-system
 ---
 apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-leader-election
+  namespace: kourier-system
+data:
+  enabledComponents: kourier
```

Comment:
Personally, it's fine with me to enable this by default; in general the convention for configmaps like this seems to be to include an explanation of what's in the file. Example from previous PR: https://github.com/knative/serving/pull/6683/files#diff-75b02e0c469b259e0eea8dd0e40c3dd7

Reply:
@pmorie please, review my latest commit. Thanks for you help enabling controller HA to kourier.